src/test-run/index.js

```diff
@@ -83,10 +103,12 @@ export default class TestRun extends Session {
         // TODO
     }
 
-    executeCommand (command) {
+    executeCommand (command, callsite) {
         if (this.pendingCommand)
             throw new Error('An attempt to execute a command when a previous command is still being executed was detected.');
 
+        this.currentCommandCallsite = callsite;
```

Comment:
BTW, why we need this var if we store callsite in `pendingCommand`? I think we should use it from there: `this.pendingCommand && this.pendingCommand.callsite`